portafly/src/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import './index.css'
```

Comment:
@didierofrivia You need to import PF4 base styles before importing any React component
```suggestion
import '@patternfly/react-core/dist/styles/base.css'
import './index.css'
```